capstone/src/main/resources/static/event.html

```diff
@@ -49,28 +43,22 @@ <h1>Create an Event </h1>
         <label for="eventDescription"> Event Description: </label> <br>
```

Comment:
Sorry I couldn't comment on a line that you didn't change, but on line 38 you don't have a counterpart for "for="eventlocation"" (would have ever noticed it but intellij caught it)